aws/rotatekey/requirements.txt

```diff
@@ -0,0 +1 @@
+boto3==1.9.49
```

Comment:
Is pinning a good idea? Often users are gonna have newer versions already installed and it'd be much more convenient if they could just run it off that.

Reply:
don't forget to change this :P so we can get the PR merged